backend/src/common/types/types.ts

```diff
@@ -7,6 +7,10 @@ export {
   type UsersByIdResponseDto,
   type UsersGetAllResponseDto,
 } from './user/user';
+export { type PermissionsGetAllResponseDto } from './permission/permission';
```

Comment:
Should this import be ./permissions/permissions' because you take type PermissionsGetAllResponseDto entity?

Reply:
types should be in singular form